dev/bin/configure.sh

```diff
@@ -97,7 +97,6 @@ function configure_kafka {
 }
 
 function configure_managed_connectors {
-  check_required_variable "MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN"
```

Comment:
can you check that `MANAGED_CONNECTORS_AUTH_CLIENT_ID` and `MANAGED_CONNECTORS_AUTH_CREDENTIALS_SECRET` are set?